icicle/appUtils/ntt/kernel_ntt.cu

```diff
@@ -462,81 +482,89 @@ namespace ntt {
     }
 
     if (log_size == 4) {
+      const int NOF_BLOCKS = (2 * batch_size + 64 - 1) / 64;
+      const int NOF_THREADS = min(64, 2 * batch_size);
```

Comment:
Small remark: imo
```c++
const int NOF_THREADS = min(64, 2 * batch_size);
const int NOF_BLOCKS = (2 * batch_size + NOF_THREADS - 1) / NOF_THREADS;
```
is better as it allows to change `NOF_THREADS` without accidentally breaking `NOF_BLOCKS` in the process. Repeats in several other places too.

Reply:
fixed